<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loginwindow</class>
 <widget class="QWidget" name="Loginwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="Designedby">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>262</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Designed by Young Inc Â© 2021 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>193</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Account">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>193</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="FogetPassword">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
   <property name="text">
    <string>Find Code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRegistAccount">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>560</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>560</width>
     <height>500</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Background/BK1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="RememberPsd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>124</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Remember</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="AutoLogin">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>122</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Auto-login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>80</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>Designedby</zorder>
  <zorder>PasswordEdit</zorder>
  <zorder>Account</zorder>
  <zorder>FogetPassword</zorder>
  <zorder>ButtonRegistAccount</zorder>
  <zorder>RememberPsd</zorder>
  <zorder>AutoLogin</zorder>
  <zorder>LoginButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
